lametro/templates/lametro/legislation.html

```diff
@@ -169,6 +168,22 @@ <h3 class="no-pad-bottom"><i class="fa fa-file-text" aria-hidden="true"></i> Rel
                 </div>
                 <div class="divider"></div>
             {% endif %}
+
+            {% if legislation.topics %}
+            <br>
+              <h3><i class="fa fa-tag" aria-hidden="true"></i> Tags</h3>
```

Comment:
should we stick with "topics" for consistency?

Reply:
good q! i tried that and it didn't work—it looks like `attachments` and `related_bills` are separate context variables. 

i'll add in a context variable for `topics` for consistency.